Wave/Wave/SingleSelectionSetting.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  SingleSelectionSetting.swift
+//  Wave
+//
+//  Created by Jimmy McDermott on 7/5/16.
+//  Copyright © 2016 BlueLine Labs. All rights reserved.
+//
+
+import Foundation
+
+public final class SingleSelectionSection: SectionType, SettingType {
+    
+    public var possibleValues: [Any]
+    public var enableCustom = false
+    public var customOption: RowType?
+    public var name: String
+    private var settings = [StringSetting]()
+    
+    public init(possibleValues: [Any], enableCustom: Bool, customOption: RowType?, name: String) {
```

Comment:
That's a good point. I guess also automatically converting it to a `StringSetting` is unfair to the developer. Should I switch on the type (right now only Int and String) and then append the setting accordingly? `settings` would have to be of type `[Setting]` which is annoying